pkg/controller/versions/strategy_v2_1.go

```diff
@@ -74,7 +74,7 @@ var (
 		},
 		MeshConfigChart: {
 			path:         "mesh-config",
-			enabledField: "",
+			enabledField: "meshConfig",
```

Comment:
hmm. this is going to be weird, as we're writing out the contents of the `meshConfig` dict into the actual meshconfig configmap. so it will now contain an `enabled: true/false`. we should avoid that IMO